receiver/jaegerreceiver/config.go

```diff
@@ -94,38 +94,36 @@ func (cfg *Config) Validate() error {
 		return fmt.Errorf("must specify at least one protocol when using the Jaeger receiver")
 	}
 
-	var grpcPort int
 	if cfg.GRPC != nil {
-		var err error
-		if grpcPort, err = extractPortFromEndpoint(cfg.GRPC.NetAddr.Endpoint); err != nil {
+		if err := checkPortFromEndpoint(cfg.GRPC.NetAddr.Endpoint); err != nil {
 			return fmt.Errorf("unable to extract port for the gRPC endpoint: %w", err)
```

Comment:
The word "extract" here doesn't make much sense anymore... not sure if it ever would have to an end user :D

Reply:
Done.